lib/currency.rb

```diff
@@ -0,0 +1,29 @@
+module Currency
+  extend self
+  
+  def args(*currency)
+    currency = currency.flatten.compact
```

Comment:
Without `flatten` everywhere there is:

```def foo(*currency)
    bar(currency)
    #...
  end```

will need to be changed to `bar(*currency)` because otherwise it will pass [[...]]